### PR TITLE
Update MetaDTA Version and Link and target _blank

### PR DESCRIPTION
**Pull request by RyanJField**
Fixes link for MetaDTA and Iframe links

### CODE REVIEW

File: ui.R [hunk 1]
```diff
@@ -56,20 +56,20 @@ tabItem(
                          tags$a(href="https://onlinelibrary.wiley.com/doi/full/10.1002/jrsm.1439", "Patel A, Cooper NJ, Freeman SC, Sutton AJ. 
                          Graphical enhancements to summary receiver operating characteristic plots to facilitate the analysis and reporting
                          of meta-analysis of diagnostic test accuracy data. Research Synthesis Methods 2020, https://doi.org/10.1002/jrsm.1439.
-                           "), 
-                         "which can be accessed at", tags$a(href="https://crsu.shinyapps.io/dta_ma/", "MetaDTA version 2.0")),
+                           ", target="_blank"), 
+                         "which can be accessed at", tags$a(href="https://crsu.shinyapps.io/MetaDTA/", "MetaDTA version 2.1.3", target="_blank")),
```
Comment (tommorris168) [inline ui.R:60]: The MetaDTA version will be 2.1.4 when you next deploy it.

**Review by tommorris168 — CHANGES_REQUESTED** [review]
Update MetaDTA version.

File: ui.R (hunk 1 above)
Comment (RyanJField) [reply ui.R:60]: Hi Tom, I didn’t bump the version as there were no changes that affected functionality, I can do though if you think we should?

Comment (tommorris168) [reply ui.R:60]: Strictly we should, but I didn't realise you were on the move when I wrote that, so for expediency I approve the rules being broken on this occasion! I'll approve this PR.

**Review by tommorris168 — APPROVED** [review]

File: ui.R (hunk 1 above)
Comment (RyanJField) [reply ui.R:60]: Thanks, did you want me to bump the MetaBayesDTA version before merging and deploying?

**Pull request merged**